bundles/net.tourbook.device.garmin.fit/src/net/tourbook/device/garmin/fit/FitData.java

```diff
@@ -438,12 +479,13 @@ private boolean finalizeTour_Type(final TourData tourData, final String parsedTo
                 TourDatabase.clearTourTypes();
                 TourManager.getInstance().clearTourDataCache();
```

Comment:
The Tour Type (Filter) List in the app UI is not updated when a new tour type is created, this can be solved with this code

```
Display.getDefault().syncExec(new Runnable() {
   public void run() {
      // fire modify event
      TourbookPlugin.getPrefStore().setValue(ITourbookPreferences.TOUR_TYPE_LIST_IS_MODIFIED, Math.random());
   }
});
```